std_msgs/Header header
int16 ch_1
int16 ch_2
int16 key_value